src/Tizen.NUI.Components/Controls/RecyclerView/Layouter/LinearLayouter.cs

```diff
@@ -642,20 +654,96 @@ protected override (int start, int end) FindVisibleItems((float X, float Y) visi
             return found;
         }
 
-        private float GetItemSize(int index)
+        // Item position excluding margins.
+        internal override (float X, float Y) GetItemPosition(int index)
+        {
+            int spaceStartX = Padding.Start;
+            int spaceStartY = Padding.Top;
+            if (colView.InternalItemSource.IsHeader(index))
+            {
+                return (spaceStartX + headerMargin.Start, spaceStartY + headerMargin.Top);
+            }
+            else if (colView.InternalItemSource.IsFooter(index))
+            {
+                return ((IsHorizontal? ScrollContentSize - footerSize - Padding.End + footerMargin.Start : spaceStartX + footerMargin.Start),
+                        (IsHorizontal? spaceStartY + footerMargin.Top : ScrollContentSize - footerSize - Padding.Bottom - footerMargin.Top));
```

Comment:
It seems that X and Y calculations are different about footerMargin.Start and footerMargin.Top.
```
+ footerMargin.Start
- footerMargin.Top
```